tests/test_panelinha_2.py

```diff
@@ -16,7 +16,7 @@ def test_author(self):
         self.assertEqual(self.harvester_class.author(), "Panelinha")
 
     def test_total_time(self):
-        self.assertEqual(0, self.harvester_class.total_time())
+        self.assertEqual(None, self.harvester_class.total_time())
```

Comment:
@jknndy unfortunately I think this edit is going to cause a conflict for one of us to resolve -- this method has been moved to an earlier line in the file in your branch, while in my branch this expected value has changed.  _Sometimes_ automated merge resolution can be clever enough to handle things like that, but in this case, I don't think it will.

Reply:
Ok! I'll take a look at it once this is merged in.

Reply:
Sounds good, thank you.  Merging this in a few moments..